games.json

```diff
@@ -8323,8 +8337,8 @@
   },
   {
     "url": "https://snowbreak.amazingseasun.com/",
-    "slug": "snowbreak-containment-zone-row%2Fintl",
-    "name": "Snowbreak: Containment Zone (ROW/INTL)",
+    "slug": "snowbreak-containment-zone-overseas",
+    "name": "Snowbreak: Containment Zone (INTL)",
```

Comment:
I don't feel like this was a necessary change. Also calling the international version the "overseas" version of the game, doesn't really make sense to me? From the perspective of a eastern Asian country, this would probably be correct.

Reply:
@Starz0r Removed "overseas" from the URL slug while reverting the name (by personal preference). Let me know if I need to change anything.